src/utils/update/review_trusts.ts

```diff
@@ -28,11 +28,11 @@ export const putTrustInReview = (appData: ApplicationData) => {
 export const hasTrusteesToReview = (trust: Trust | undefined, trusteeType: string) => {
   switch (trusteeType) {
       case TrusteeType.HISTORICAL:
-        return !trust?.review_status?.reviewed_former_bos && (trust?.HISTORICAL_BO ?? []).length > 0;
+        return !(trust?.review_status?.reviewed_former_bos ?? true) && (trust?.HISTORICAL_BO ?? []).length > 0;
```

Comment:
safer to assume undefined is not reviewed? "?? false" instead of "?? true"

Reply:
I can't recall my reason for adding this at the time, so I'll revert this change for now. If it comes back to me I can reapply it in another PR.